mapping/adminset.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+module Mapping
+  class AdminSet
+    require 'csv'
+    require 'yaml'
+
+    extend Extensions::BasicValueHandler
+
+    ##
+    # Locate adminset with the passing owning_collection handle
+    #
+    # @param[String] value - handle of owning collection
+    # @param[Array] *args - properties passed to the method
+    # return - adminset_id, a blank adminset_id maps to 'Default' adminset in Hyrax 
+    def self.lookup_admin_set(value, *args)
+      csv_file = File.join(File.dirname(__FILE__), '../lookup/collectionlist.csv')
+      lines = CSV.read(csv_file, :headers=>true).map(&:to_hash)
+      lines.each do |line|
```

Comment:
the `lines.each....` can both be simplified and adjusted so use case-insensitive comparison for the value matching collection_handle:

`lines.map { |line| line[:Admin_Set] if line[:collection_handle].casecmp(value).zero?`

`#map` will return an array of `line[:Admin_Set]` so long as the `line[:collection_handle]` matches `value` regardless of the case of characters..